wallet/storage/src/lib.rs

```diff
@@ -69,8 +72,12 @@ pub trait WalletStorageRead {
 pub trait WalletStorageWrite: WalletStorageRead {
     /// Set storage version
     fn set_storage_version(&mut self, version: u32) -> Result<()>;
-    fn set_utxo(&mut self, outpoint: &AccountOutPointId, entry: Utxo) -> Result<()>;
-    fn del_utxo(&mut self, outpoint: &AccountOutPointId) -> Result<()>;
+    fn set_block(
+        &mut self,
+        block_height: &AccountBlockHeight,
+        block: &WalletBlock,
+    ) -> crate::Result<()>;
```

Comment:
Why can't we retrieve this information from the node? This can be too much for a wallet.

Reply:
We only store kernel inputs and reward outputs here (and only blocks that are relevant to the wallet). So it's not much different from storing `WalletTx`.